handler/src/main/java/io/netty/handler/ssl/PemValue.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.util.AbstractReferenceCounted;
+import io.netty.util.IllegalReferenceCountException;
+
+/**
+ * A PEM encoded value.
+ *
+ * @see PemEncoded
+ * @see PemPrivateKey#toPEM(ByteBufAllocator, boolean, java.security.PrivateKey)
+ * @see PemX509Certificate#toPEM(ByteBufAllocator, boolean, java.security.cert.X509Certificate[])
+ */
+class PemValue extends AbstractReferenceCounted implements PemEncoded {
```

Comment:
Actually that will not work, `DefaultByteBufHolder#release()` returns true/false after it has released the underlying ByteBuf at which point it's no longer possible for me to wipe the bytes.